app/views/user.py

```diff
@@ -209,3 +209,21 @@ def reset_users_data_usage(db: Session = Depends(get_db),
     dbadmin = crud.get_admin(db, admin.username)
     crud.reset_all_users_data_usage(db=db, admin=dbadmin)
     return {}
+
+@app.get("/api/user/{username}/usage", tags=['User'], response_model=UserUsagesResponse)
+def get_user(username: str,
+             db: Session = Depends(get_db),
+             admin: Admin = Depends(Admin.get_current)):
+    """
+    Get users usage
+    """
+    dbuser = crud.get_user(db, username)
+    if not dbuser:
+        raise HTTPException(status_code=404, detail="User not found")
+
+    if not (admin.is_sudo or (dbuser.admin and dbuser.admin.username == admin.username)):
```

Comment:
I'm not sure why that check is necessary. could you give me some information why this is important to be a sudo here?

Reply:
😂😂，it my faults, I copy the code of get_user. so it may be not need here